docker/openemr/7.0.2/openemr.sh

```diff
@@ -24,8 +24,26 @@ swarm_wait() {
 auto_setup() {
     prepareVariables
 
-    chmod -R 600 .
-    php auto_configure.php -f ${CONFIGURATION} || return 1
+    #create temporary file cache directory for auto_configure.php to use
+    TMP_FILE_CACHE_LOCATION="/tmp/php-file-cache"
+    mkdir $TMP_FILE_CACHE_LOCATION
+
+    #create auto_configure.ini to be able to leverage opcache for operations
+    touch auto_configure.ini
+    echo "opcache.enable=1" >> auto_configure.ini
+    echo "opcache.enable_cli=1" >> auto_configure.ini
+    echo "opcache.file_cache=$TMP_FILE_CACHE_LOCATION" >> auto_configure.ini
+    echo "opcache.file_cache_only=1" >> auto_configure.ini
+    echo "opcache.file_cache_consistency_checks=1" >> auto_configure.ini
+    echo "opcache.enable_file_override=1" >> auto_configure.ini
+    echo "opcache.max_accelerated_files=1000000" >> auto_configure.ini
+
+    #run auto_configure
+    php auto_configure.php -c auto_configure.ini -f ${CONFIGURATION} || return 1
+
+    #remove temporary file cache directory and auto_configure.ini
+    rm -r $TMP_FILE_CACHE_LOCATION
+    rm auto_configure.ini
```

Comment:
also neat stuff! (very curious to see how much faster the config happens)

Reply:
Thanks! It's a really considerable speed increase. Around ~%40 runtime reduction although I'd be really curious to see what others observe.